main/templates/base.html

```diff
@@ -36,10 +36,10 @@
 				<ul id="navbar" class="nav navbar-nav collapse navbar-collapse navbar-right">
 					<!-- <li><a href="{% url 'home' %}">HOME</a></li> -->
 					<li><a href="{% url 'our_team' %}">OUR TEAM</a></li>
-					<li><a href="{% url 'projects' %}">PROJECTS</a></li>
-					<li><a target="_blank" href="https://medium.com/@EnactusUW">BLOG</a></li>
-					<!-- <li><a href="{% url 'sponsors' %}">SPONSORS</a></li> -->
+					<li><a href="/#projects">PROJECTS</a></li>
+					<!-- <li><a href="/#sponsors">SPONSORS</a></li> -->
```

Comment:
I recognize that this line is commented for now, but it's still wrong, same as the comment above.